src/app/main.ml

```diff
@@ -198,7 +200,8 @@ let entry_point = let open Gandalf_args in
         $ log_plugged_action
         $ log_everything_else
         $ status_server_port
-        $ ignore_all_unix_errors)
+        $ ignore_all_unix_errors
```

Comment:
I didn't understand what to change? I need to delete whitespace? or something else?